tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pathlib
+
+import pytest
+
+from clldutils.path import copytree
+
+
+@pytest.fixture
+def tmppath(tmpdir):
+    return pathlib.Path(str(tmpdir))
```

Comment:
For python >=3.6, `pytest`'s [`tmp_path` fixture](https://docs.pytest.org/en/stable/tmpdir.html) is exactly that. So if we don't want to support py35 (and I think we don't), this can be replaced.